truffle-config.js

```diff
@@ -6,6 +6,7 @@ const realm = process.env.WITNET_EVM_REALM ? process.env.WITNET_EVM_REALM.toLowe
 
 module.exports = {
   build_directory: `./build/${realm}/`,
+  contractrs_directory: "./contracts/",
```

Comment:
I guess this is a typo?

```suggestion
  contracts_directory: "./contracts/",
```

Reply:
Absolutely